source/quick-reference.txt

```diff
@@ -0,0 +1,267 @@
+.. _entity-framework-quick-reference:
+
+===============
+Quick Reference
+===============
+
+This page shows the provider syntax for several commands and links to
+their related API documentation.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Command
+     - Syntax
+
+   * - | **Set up DBContext**
+       |
+       | `API Documentation <https://learn.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.dbcontext>`__
```

Comment:
Q: The other Quick Ref pages also link to Usage Examples and Fundamentals. What's a good way to make note to come back for those here?

Reply:
I'll make a ticket to come back and update this page after those are done